kingpin/actors/aws/iam.py

```diff
@@ -27,6 +27,7 @@
 from kingpin.actors.aws import base
 from kingpin.actors import exceptions
 from kingpin.constants import REQUIRED
+from kingpin.constants import STATE
```

Comment:
I have a thought on how to re-factor this so its more generic, and easier for other actors to use in the future. We can chat on Monday about it. Until then, ignore this code.